records_mover/session.py

```diff
@@ -177,6 +187,10 @@ def __init__(self,
         if default_gcp_creds_name is PleaseInfer.token:
             default_gcp_creds_name = _infer_default_gcp_creds_name(session_type)
 
+        # TODO: Move this logic to creds interface
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker